src/odemis/driver/hamamatsurx.py

```diff
@@ -597,6 +598,7 @@ def __init__(self, name, role, parent, daemon=None, **kwargs):
         # [Volt] Input and indication of the trigger level for the vertical sweep.
         parent.DevParamSet(self.location, "Trig. level", 1)  # TODO check what value needed regarding HW
         parent.DevParamSet(self.location, "Trig. slope", "Rising")
+        parent.DevParamSet(self.location, "Shutter", "Closed")
```

Comment:
This needs to work with the old hardware too, which doesn't have a Shutter (controllable in software).  I guess in this case, the shutter parameter will not be available. Please check with the (old) simulator. Ideally, the driver should detect the Shutter is not controllable, and just not provide a VA.